bump-orbs.sh

```diff
@@ -14,6 +14,11 @@ if [ "$#" -gt 1 ]; then
     export CIRCLECI_CLI_TOKEN="$2"
 fi
 
+IGNORED_ORBS=""
+if [ "$#" -gt 2 ]; then
+    IGNORED_ORBS="$3"
+fi
```

Comment:
It might be time for actual arguments in this script now, seeing as arg 2 (CLI token) is supposed to be optional, and so is this. :/

Reply:
I think this works. https://github.com/eddsteel/bump-orb/pull/2/commits/ae30e853444c7e20b5a4272694697c5ef342b549

It makes sure that even when optional args aren't supplied, they will pass an empty string `''` to the script.